CONTRIBUTING.md

```diff
@@ -74,49 +74,31 @@ Build the SDK:
 npm run build
 ```
 
-> There used to be a means of building individual packages, but they now build quickly enough that there's no need.
-
 ### Running Tests
 
-#### Run All Tests
-
-```bash
-npm test
-```
-
-#### Run Unit Tests
-
-Handy during early plugin development when you can write a bunch of unit tests.
-
-```bash
-npm test -- --package PACKAGENAME --unit
-```
-
-### Run unit tests in watch mode
+`npm test` is the entrypoint to our test runner, but its not practical to use without parameters; the full suite would take over two hours to run and cross talk would probably cause tests to break each other.
 
-OK, this one's a handful and requires a global package, but there were too many possible variants to
-hardcode it any where.
+> Get the full test-runner docs via `npm test -- --help`.
 
-```bash
-npm install -g nodemon
-nodemon -w packages/PACKAGENAME/src -w packages/PACKAGENAME/test -x "npm test -- --package PACKAGENAME --node"
-```
+A local development flow might look like
 
-#### Run Node.js Tests
+1. Edit source code in `MYPACKAGE`.
+2. Use `npm run build` to build all packages .
+3. Use `npm test -- --package MYPACKAGE --node` to run the tests for just that package only in nodejs (Usually, we don't need to test both in node and the browser during development).
+4. Repeats steps 1-3 until the tests pass.
 
-Usually faster, and can build on the fly, thus no need to rebuild everything between test runs
+`npm run build` is a bit tedious when making lots of changes, so instead, we can use `npm run distsrc` to point each package's `main` entry at the raw src and let `babel-node` compile on the fly.
 
-```bash
-npm test -- --package PACKAGENAME --node
-```
+1. At the start of development, run `npm run distsrc` once.
+2. Edit source code in `MYPACKAGE`.
+3. Use `npm test -- --package MYPACKAGE --node` to run the tests for just that package only in nodejs.
+4. Repeat steps 2-3 until the tests pass. 
```

Comment:
Want to add a line to run `npm run srcdist` to revert the package.json changes back so they don't get committed?

Reply:
👍